client/js/table-view.js

```diff
@@ -2,97 +2,131 @@ const { getLetterRange } = require('./array-util');
 const { removeChildren, createTR, createTH, createTD } = require('./dom-util');
```

Comment:
The files above got fixed, but this file still has 4-space indentation. Please fix throughout.